ros/lib/config.py

```diff
@@ -141,6 +141,7 @@ def build_endpoint_url(ep):
 DAYS_UNTIL_STALE = int(os.getenv("DAYS_UNTIL_STALE", '45'))
 CW_LOGGING_FORMAT = '%(asctime)s - %(levelname)s  - %(funcName)s - %(message)s'
 ROS_PROCESSOR_PORT = int(os.getenv("ROS_PROCESSOR_PORT", "8000"))
+ROS_SUGGESTIONS_ENGINE_PORT = int(os.getenv("ROS_SUGGESTIONS_ENGINE_PORT", "8000"))
```

Comment:
hmm, I dont remember if we decided to share the 8000 as port or use new one?

Reply:
We have set the same default port for both the processor and the API, so we will be doing the same for this service as well. However, when running locally, you must have all services running on different ports.